ts_src/ethereum/index.ts

```diff
@@ -0,0 +1,253 @@
+import { AlchemyProvider, Block, FeeData, FixedNumber, HDNodeWallet, Log, ethers, Transaction, TransactionResponse, Filter, Wallet } from 'ethers';
+
+require('dotenv').config();
```

Comment:
this require should be in the example and in the ethereum file, the integrator should decide how the values are passed or stored